spec/rails_helper.rb

```diff
@@ -6,6 +6,11 @@
 require 'spec_helper'
 require 'rspec/rails'
 require 'shoulda-matchers'
+require 'capybara/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.